### PR TITLE
fix: monitor to page relation including form defaults

### PR DESCRIPTION
**Pull request by mxkaske**
- fix default values for the `defaultValue?.monitors` checkboxes
- fix `NaN` and add `EmptyState` to status page
- add `{where: monitors: {} }` relational query for the `getPagesByWorkspaces`
- fix all possible delete/update relations for the many-to-many relation `monitorsToPages`

### CODE REVIEW

File: packages/db/src/schema/page.ts [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const page = sqliteTable("page", {
 });
 
 export const pageRelations = relations(page, ({ many, one }) => ({
-  monitorsToPages: many(monitorsToPages),
+  monitors: many(monitorsToPages),
```
Comment (mxkaske) [inline packages/db/src/schema/page.ts]: I'd suggest to use `monitors`. This fits well for the relational queries from drizzle. Than, we don't have to access the values via `page.monitorsToPages`.

File: packages/api/src/router/page.ts [hunk 2]
```diff
@@ -129,16 +130,19 @@ export const pageRouter = createTRPCRouter({
...
-          ),
-        )
-        .all();
+      return opts.ctx.db.query.page.findMany({
+        where: and(
+          eq(page.workspaceId, opts.input.workspaceId),
+          inArray(page.workspaceId, workspaceIds),
+        ),
+        with: {
+          monitors: {
```
Comment (thibaultleouay) [inline packages/api/src/router/page.ts]: Does it really work because the relation is on pagesToMonitors ?

Comment (mxkaske) [reply packages/api/src/router/page.ts]: I renamed `monitorsToPages` to `monitors`. Just to use monitors all over the frontend. tbd.

File: packages/db/src/schema/page.ts (hunk 1 above)
Comment (thibaultleouay) [reply packages/db/src/schema/page.ts]: but it's a many-to-many relation ship we need a join table

Comment (thibaultleouay) [reply packages/db/src/schema/page.ts]: https://orm.drizzle.team/docs/rqb#many-to-many 😭

Comment (mxkaske) [reply packages/db/src/schema/page.ts]: Thanks a lot! Reverted back! 🚀

**Pull request merged**